src/js/index.js

```diff
@@ -25,7 +25,7 @@ import { css } from 'styled-components';
 import { deepFreeze } from 'grommet/utils/object';
 import { normalizeColor } from 'grommet/utils/colors';
 import { parseMetricToNum } from 'grommet/utils/mixins';
-import { FormNext, FormPrevious, Clear } from 'grommet-icons';
+import { Clear } from 'grommet-icons';
```

Comment:
Use clear icon from mnet icons
```suggestion
import { Clear } from 'grommet-icons';
```